views/secretWord.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
And here, once you have your head.ejs partial, you just include it.  Then, you include the head.ejs partial, to show errors or messages as needed.  Then you continue with the rest of the body, down to where you would put the </body> tag.  But you don't put that in secretWord.ejs.  Instead you include the footer.ejs.